### PR TITLE
Update overlays for galaxy A32 and add system ui overlays.

### PR DESCRIPTION
**Pull request by realahnet**
I made new system ui overlays that actually work! and new configs. Tested the overlays out. they work great.

### CODE REVIEW

**Comment by SysWhiteDev** [comment]
how did you check if the overlay was working? because i have no idea how

**Comment by realahnet** [comment]
> how did you check if the overlay was working? because i have no idea how

i pushed the built apks to my /system/product they seemed to be working and i didnt see any errors in adb logcat. i didnt see any errors about the extra properties that do not exist. so they work jsut fine.

**Comment by SysWhiteDev** [comment]
sorry if i wasn't clear, but i mean what should the phone do differently if it is really working?

**Comment by realahnet** [comment]
> sorry if i wasn't clear, but i mean what should the phone do differently if it is really working?

i fixed some values which did make an effect. the brightness slider was working way more accurately. i also added a config for biometric sensors because on new td patches fp sensor was being disabled for most users because it allowed pixel devices to boot. this is to compensate for that. i would like to have that in there incase something like this happens again. this fixed it on ricedroid gsi. so there are some changes and system ui overlay also helped! that status bar potrait config was also fixed some wierd placement there. so yes i did not just add unncessary configs. they all help.

**Comment by realahnet** [comment]
one more thing. just noticed pushing my new overlays into some gsi's fixes face unlock somehow.

File: Samsung/A32-SystemUI/res/values/dimens.xml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <item type="dimen" name="config_screenBrightnessDimFloat">0.047058824</item>
```
Comment (phhusson) [inline Samsung/A32-SystemUI/res/values/dimens.xml]: that's not a sysui prop...

File: Samsung/A32-SystemUI/res/values/bools.xml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <bool name="config_enableWifiDisplay">true</bool>
+    <bool name="config_adaptive_display_solution_enabled">true</bool>
+    <bool name="config_all_around_sensing_enabled">false</bool>
+    <bool name="config_big_data_logging_enabled">false</bool>
+    <bool name="config_dynamic_automatic_brightness_available">true</bool>
```
Comment (phhusson) [inline Samsung/A32-SystemUI/res/values/bools.xml]: that's not in systemui

**Review by phhusson — CHANGES_REQUESTED** [review]
You're confusing systemui and frameworks-res overlay...

File: Samsung/A32-SystemUI/res/values/dimens.xml (hunk 1 above)
Comment (realahnet) [reply Samsung/A32-SystemUI/res/values/dimens.xml]: im sorry i wasnt aware. i am not an expert with overlays. i'll remove them

File: Samsung/A32-SystemUI/res/values/bools.xml (hunk 2 above)
Comment (realahnet) [reply Samsung/A32-SystemUI/res/values/bools.xml]: okay i'll remove those. sorry i am not an expert in overlays

File: Samsung/A32-SystemUI/res/values/dimens.xml (hunk 1 above)
Comment (realahnet) [reply Samsung/A32-SystemUI/res/values/dimens.xml]: configs removed

File: Samsung/A32-SystemUI/res/values/bools.xml (hunk 2 above)
Comment (realahnet) [reply Samsung/A32-SystemUI/res/values/bools.xml]: configs removed

File: Samsung/A32-SystemUI/res/values/bools.xml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
```
Comment (phhusson) [inline Samsung/A32-SystemUI/res/values/bools.xml:2]: none of those bools are in sysui

File: Samsung/A32-SystemUI/res/values/dimens.xml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <item type="dimen" name="config_screenBrightnessSettingDefaultFloat">0.5019608</item>
```
Comment (phhusson) [inline Samsung/A32-SystemUI/res/values/dimens.xml:3]: none of the config_xxxx are in sysui

**Review by phhusson — CHANGES_REQUESTED** [review]

File: Samsung/A32-SystemUI/res/values/bools.xml (hunk 3 above)
Comment (realahnet) [reply Samsung/A32-SystemUI/res/values/bools.xml:2]: i dont understand. does that mean my whole system ui overlay is useless?

File: Samsung/A32-SystemUI/res/values/dimens.xml (hunk 4 above)
Comment (realahnet) [reply Samsung/A32-SystemUI/res/values/dimens.xml:3]: so that means my whole system ui config is useless?

Comment (phhusson) [reply Samsung/A32-SystemUI/res/values/dimens.xml:3]: no you can keep the one not starting with config_

Comment (realahnet) [reply Samsung/A32-SystemUI/res/values/dimens.xml:3]: oh okay. btw i am not exprienced with overlays. but is there a list of configs or something where i could know what all the configs are? i followed the guide and thats how i made these 2 overlays.

**Pull request closed without merging**